sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java

```diff
@@ -283,6 +283,11 @@ private View prepareRootView(@NonNull LayoutInflater inflater,
                 return true;
             }
 
+            final boolean ignoreBackPress = Boolean.FALSE.equals(config.getStopVerificationOnBackPress());
+            if (ignoreBackPress) {
+                return true;
+            }
+
             return webViewHelper != null && webViewHelper.shouldRetry(
```

Comment:
Can't `config.getStopVerificationOnBackPress()` already be achieved with a custom `config.retryPredicate`?